app/models/character-store/character-store.test.ts

```diff
@@ -0,0 +1,7 @@
+import { CharacterStoreModel } from "./character-store"
```

Comment:
Do these character / character store models come with Ignite now? We can probably remove them from this app. I can make a note in the blog post text

Reply:
After I opened this PR I realize that. Yes, those come with Ignite. I will delete that too. 🙂